test/basicfuns.jl

```diff
@@ -468,3 +468,93 @@ end
     @test cexpexp(-Inf) == 0.0
     @test cexpexp(0) == (ℯ - 1) / ℯ
 end
+
+@testset "loglogistic: $T" for T in (Float16, Float32, Float64)
+    lim1 = T === Float16 ? -14.0 : -50.0
+    lim2 = T === Float16 ? -10.0 : -37.0
+    xs = T[Inf, -Inf, 0.0, lim1, lim2]
+    for x in xs
+        @test loglogistic(x) == log(logistic(x))
```

Comment:
If we test multiple types `T`, I think it would be good to also check that - as desired - the results are of type `T`.

Reply:
Anything else, @devmotion?